vignettes/Moonlight2R.Rmd

```diff
@@ -1,1136 +1,908 @@
+  DOI: "10.1016/j.ccell.2016.04.002"
+  volume: 29
+  pages: 43-49
+  issued:
+    year: 2016 
+
+- id: ref23
+  title: Complex heatmaps reveal patterns and correlations in multidimensional genomic data
+  author: 
+  - family: Gu, Zuguang and Eils, Roland and Schlesner, Matthias
+  given:
+  journal: Bioinformatics
+  URL: "http://dx.doi.org/10.1016/j.ccell.2016.04.002"
+  DOI: "10.1016/j.ccell.2016.04.002"
+  pages: "btw313"
+  issued:
+   year: 2016 
+
+- id: ref24
+  title: "TCGA Workflow: Analyze cancer genomics and epigenomics data using Bioconductor packages"
+  author: 
+  - family:  Silva, TC and Colaprico, A and Olsen, C and D'Angelo, F and Bontempi, G and Ceccarelli, M and Noushmehr, H
+  given:
+  journal: F1000Research
+  URL: "http://dx.doi.org/10.12688/f1000research.8923.1"
+  DOI: "10.12688/f1000research.8923.1"
+  volume: 5
+  number: 1542
+  issued:
+   year: 2016 
+
+- id: ref25
+  title: "TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data"
+  author: 
+  - family:  Colaprico, Antonio and Silva, Tiago C. and Olsen, Catharina and Garofano, Luciano and Cava, Claudia and Garolini, Davide and Sabedot, Thais S. and Malta, Tathiane M. and Pagnotta, Stefano M. and Castiglioni, Isabella and Ceccarelli, Michele and Bontempi, Gianluca and Noushmehr, Houtan
+  given:
+  journal: Nucleic Acids Research
+  URL: "http://dx.doi.org/10.1093/nar/gkv1507"
+  DOI: "10.1093/nar/gkv1507"
+  volume: 44
+  number: 8
+  pages: e71
+  issued:
+   year: 2016 
+
+- id: ref26
+  title: "Interpreting pathways to discover cancer driver genes with Moonlight"
+  author: 
+  - family:  Colaprico, Antonio and Olsen, Catharina and Bailey, Matthew H. and Odom, Gabriel J. and Terkelsen, Thilde and Silva, Tiago C. and Olsen, André V. and Cantini, Laura and Zinovyev, Andrei and Barillot, Emmanuel and Noushmehr, Houtan and Bertoli, Gloria and Castiglioni, Isabella and Cava, Claudia and Bontempi, Gianluca and Chen, Xi Steven and Papaleo, Elena
+  given:
+  journal: Nature Communications
+  URL: "https://doi.org/10.1038/s41467-019-13803-0"
+  DOI: "10.1038/s41467-019-13803-0"
+  volume: 11
+  number: 69
+  issued:
+   year: 2020
+
+- id: ref27
+  title: "An Automatized Workflow to Study Mechanistic Indicators for Driver Gene Prediction with Moonlight"
+  author: 
+  - family:  Saksager, Astrid and Nourbakhsh, Mona and Tom, Nikola and Chen, Xi Steven and Colaprico, Antonio and Olsen, Catharina and Tiberti, Matteo and Papaleo, Elena
+  given:
+  journal: bioRxiv
+  URL: "https://www.biorxiv.org/content/10.1101/2022.11.18.517066v1"
+  DOI: "10.1101/2022.11.18.517066"
+  issued:
+   year: 2022
+
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  \usepackage[utf8]{inputenc}
+---
+  
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(dpi = 300)
+knitr::opts_chunk$set(cache=FALSE)
+```
+
+```{r, echo = FALSE,hide=TRUE, message=FALSE,warning=FALSE}
+library(dbplyr)
```

Comment:
from my test it looks like we don't need this anymore after getting rid of TCGAbiolinks - the system built package/vignettes without issues. I think it can be removed

Reply:
removed